packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -147,7 +147,10 @@ async function instantiateModule(
         context,
         urlStack,
         fixStacktrace,
-      )
+      ).catch((err) => {
+        err.importee = dep
+        throw err
+      })
```

Comment:
I'm a bit confused about this part, aren't we already attaching the importee at

https://github.com/vitejs/vite/blob/9697e641f9feb9e2d31f52b6add2648b88515425/packages/vite/src/node/ssr/ssrModuleLoader.ts#L269-L270

Wouldn't this `dep` variable be the `importer` instead?

Reply:
Yeah I realized now that we only handled `nodeImport` and not other imports. This `catch` looks fine to me then, but would be good to have a comment as well.

Reply:
I hoisted the catch error logic to the `ssrImport`